isis/src/base/objs/UserInterface/UserInterface.cpp

```diff
@@ -517,6 +530,9 @@ namespace Isis {
         throw IException(e, IException::User, "Invalid command line", _FILEINFO_);
       }
 
+    } 
+    if( usedDashLast == true){  
```

Comment:
I'm not sure that this results in the intended behavior.  From my testing, this prints out the string "<program_name> -last <new options>" instead of expanding -last into the previously specified arguments.

I ran:
$> lrowac2isis from= wac0000a1c4.img to= out.cub
$> lrowac2isis -last to=out2.cub
and got the output:
lrowac2isis -last to=out2.cub

I would expect to get the output:
lrowac2isis from=wac0000a1c4.img to=out2.cub

@scsides Can you confirm that my understanding of the intended behavior is correct?

Reply:
your expectations are correct